File: chambainfo-backend/chambainfo-backend/src/main/java/com/chambainfo/backend/dto/PostulacionResponse.java
package com.chambainfo.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PostulacionResponse {

    private Long id;
    private Long empleoId;
    private String nombreEmpleo;
    private Long trabajadorId;
    private String trabajadorNombre;
    private String trabajadorDni;
    private String trabajadorCelular;
    private String mensaje;
    private String estado;
    private LocalDateTime fechaPostulacion;
}

File: chambainfo-front/app/src/main/java/com/chambainfo/app/data/model/PostulacionResponse.kt
package com.chambainfo.app.data.model

data class PostulacionResponse(
    val id: Long,
    val empleoId: Long,
    val nombreEmpleo: String,
    val trabajadorId: Long,
    val trabajadorNombre: String,
    val trabajadorDni: String,
    val trabajadorCelular: String,
    val mensaje: String,
    val estado: String,
    val fechaPostulacion: String
)